fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -39,54 +41,54 @@
 /**
  * This class is a servlet filter which is registered with the REST API's servlet. The main purpose of the class is to
  * log entry/exit information and elapsed time for each REST API request processed by the server.
- * 
- * @author padams
  */
-public class FHIRRestServletFilter implements Filter {
+public class FHIRRestServletFilter extends HttpFilter {
+    private static final long serialVersionUID = 1L;
+
     private static final Logger log = Logger.getLogger(FHIRRestServletFilter.class.getName());
 
     private static String tenantIdHeaderName = null;
     private static String datastoreIdHeaderName = null;
+    private static String originalRequestUriHeaderName = null;
```

Comment:
I thought Filters were stateless? ... so the reason I mention this... 
if the configuration changes, a server restart needs to be called to re-activate the init filter

Reply:
maybe document that behavior?

Reply:
added to section 5.1.3 of the userguide